src/main/java/org/jboss/qa/KarafClientMojo.java

```diff
@@ -220,10 +220,7 @@ public void welcome(String banner) {
 			// Expects issue KARAF-2623 is fixed
 			final boolean isError = (channel.getExitStatus() != null && channel.getExitStatus().intValue() != 0);
 			if (isError) {
-				final String errorMarker = Ansi.ansi().fg(Color.RED).toString();
-				final int fromIndex = sout.toString().indexOf(errorMarker) + errorMarker.length();
-				final int toIndex = sout.toString().lastIndexOf(Ansi.ansi().fg(Color.DEFAULT).toString());
-				throw new MojoExecutionException(NEW_LINE + sout.toString().substring(fromIndex, toIndex));
```

Comment:
Problem has been there when no markers was there:
fromIndex=4, toIndex=-1 - this leads to index out of bound exception